api/handlers/node.go

```diff
@@ -65,6 +96,72 @@ func (h *Node) Identity(w http.ResponseWriter, r *http.Request) error {
 
 func (h *Node) Peers(w http.ResponseWriter, r *http.Request) error {
 	peers := h.Network.Peers()
+	resp := h.peers(peers)
+	return api.Render(w, r, resp)
+}
+
+func (h *Node) Topics(w http.ResponseWriter, r *http.Request) error {
+	allpeers, peerbytpc := h.TopicIndex.PeersByTopic()
+	alland := AllPeersAndTopicsJSON{}
+	tpcs := []topicIndexJSON{}
+	for topic, peers := range peerbytpc {
+		tpcs = append(tpcs, topicIndexJSON{TopicName: topic, Peers: peers})
+	}
+	alland.AllPeers = allpeers
+	alland.PeersByTopic = tpcs
+
+	return api.Render(w, r, alland)
+}
+
+func (h *Node) Health(w http.ResponseWriter, r *http.Request) error {
+	ctx := context.Background()
+	resp := healthCheckJSON{
+		BeaconConnectionHealthStatus:    good.String(),
+		ExecutionConnectionHealthStatus: good.String(),
+		EventSyncHealthStatus:           good.String(),
+		PeersHealthStatus:               good.String(),
+	}
+	// Check ports being used.
+	addrs := h.Network.ListenAddresses()
+	for _, addr := range addrs {
+		if addr.String() == "/p2p-circuit" || addr.Decapsulate(multiaddr.StringCast("/ip4/0.0.0.0")) == nil {
+			continue
+		}
+		resp.LocalPortsListening = addr.String()
+	}
+	// Performing various health checks.
+	resp.BeaconConnectionHealthStatus = performHealthCheck(h.NodeProber.CheckBeaconNodeHealth, ctx)
+	resp.ExecutionConnectionHealthStatus = performHealthCheck(h.NodeProber.CheckExecutionNodeHealth, ctx)
+	resp.EventSyncHealthStatus = performHealthCheck(h.NodeProber.CheckEventSyncerHealth, ctx)
+	// Check peers connection.
+	var activePeerCount int
+	peers := h.Network.Peers()
+	for _, p := range h.peers(peers) {
+		if p.Connectedness == "Connected" {
+			activePeerCount++
+		}
+	}
+	switch {
+	case activePeerCount > 0 && activePeerCount < healthyPeersAmount:
+		resp.PeersHealthStatus = fmt.Sprintf("%s: %d peers are connected", bad, activePeerCount)
+	case activePeerCount == 0:
+		resp.PeersHealthStatus = fmt.Sprintf("%s: %s", bad, "error: no peers are connected")
```

Comment:
This can be simplified to
```suggestion
		resp.PeersHealthStatus = fmt.Sprintf("%s: error: no peers are connected", bad)
```